plugins/base/base-test-utils/src/main/kotlin/renderers/defaultSourceSet.kt

```diff
@@ -1,30 +0,0 @@
-package renderers
```

Comment:
I think this file should be removed but looks like git sees it as empty

Reply:
This file was removed, wan't it?
Cant find it locally anymore and Git tells me I am up to date with origin 🤔

Reply:
Oh yes, I wasn't, everything is ok ☺️